### ISSUE TITLE
[New Functionality] Task Mining

### ISSUE THREAD

**Issue by cobusvanzylsmit** [saucepleez/taskt#260]
Actual task mining solutions are quite rare, very new and quite expensive. How about recording tasks with Taskt recorder accross a variety of machines where people conduct similar activties and collate the results to determine automation potential. This is a game changer for identifying RPA opportunities.

**Comment by saucepleez** [comment]
This is true! I have thought about this for a while now.  I am not sure whether it is better to bundle this type of functionality with this existing project or start a new one entirely.
